index.js

```diff
@@ -1,63 +1,67 @@
-// Make sure if the local storage in empty than add an empty array
-if (localStorage.getItem('Added Books') == null) {
-  localStorage.setItem('Added Books', JSON.stringify([]));
-}
+class BookCollection {
+  // Make sure if the local storage in empty than add an empty array
+  static getStoredBooks() {
+    if (localStorage.getItem('Added books') === null) {
+      localStorage.setItem('Added books', JSON.stringify([]));
+    }
+    return JSON.parse(localStorage.getItem('Added books'));
+  }
+
+  // Store the books data into the local storage
+  static updateStoredBooks(books) {
+    localStorage.setItem('Added books', JSON.stringify(books));
+  }
+
+  static addNewBook(bookTitle, bookAuthor) {
+    const storedBooks = BookCollection.getStoredBooks();
+    const newBook = {
+      title: bookTitle,
+      author: bookAuthor,
+    };
+    storedBooks.push(newBook);
+    BookCollection.updateStoredBooks(storedBooks);
+    BookCollection.displayBooks(storedBooks);
+  }
```

Comment:
Same as the previews comment this method is meant to be adding to the new instance of books , and by adding the constructor you can now access the books by using the key word this 

``this.books....``